applications/asset_tracker_v2/src/modules/data_module.c

```diff
@@ -982,28 +982,45 @@ static void agps_request_handle(struct nrf_modem_gnss_agps_data_frame *incoming_
 	/* If CONFIG_NRF_CLOUD_MQTT is enabled, the nRF Cloud MQTT transport library will be used
 	 * to send the request.
 	 */
```

Comment:
This comment would be better inside the `CONFIG_NRF_CLOUD_MQTT` flagged section.